assets/base.css

```diff
@@ -3200,7 +3200,6 @@ details-disclosure > details {
   .animate--hover-3d-lift .deferred-media .deferred-media__poster-button {
     transition: transform var(--duration-long) ease, box-shadow var(--duration-long) ease;
     transform-origin: center;
-    z-index: 1;
```

Comment:
@ludoboludo Just a heads up: I think we're ok removing this rule entirely for now. We're overriding it for cards with the new changes below, and I don't think it was actually necessary for buttons anyway — they appear to work ok in my testing as-is.

Reply:
This will fix the 3D Lift portion of #2596.